### PR TITLE
Change format of sub-module names in the object module to str

### PR DESCRIPTION
**Pull request by nsundareswaran**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @nsundareswaran, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: boxsdk/object/__init__.py [hunk 1]
```diff
@@ -2,19 +2,6 @@
 
 from __future__ import unicode_literals
 
-import six
 
+__all__ = list(map(str, ['collaboration', 'events', 'file', 'folder', 'group', 'group_membership', 'search', 'user']))
```
Comment (jmoldow) [inline boxsdk/object/__init__.py]: Delete this blank line to make pep8 happy.

File: boxsdk/object/__init__.py [hunk 2]
```diff
@@ -2,19 +2,6 @@
...
```
Comment (jmoldow) [inline boxsdk/object/__init__.py]: You can fix this with either

    from six.moves import map  # pylint:disable=redefined-builtin

or

    __all__ = [str(s) for s in ['collaboration', 'events', 'file', 'folder', 'group', 'group_membership', 'search', 'user']]

**Comment by jmoldow** [comment]
:+1:

**Comment by nsundareswaran** [comment]
CLA Signed

**Comment by boxcla** [comment]
Verified that @nsundareswaran has just signed the CLA. Thanks, and we look forward to your contribution.

**Pull request merged**